File: intro-to-akka-demo-scala/src/main/scala/be/yannickdeturck/akkademo/pizzaorderer/WorkshopRunner.scala
package be.yannickdeturck.akkademo.pizzaorderer

import akka.actor.ActorSystem
import be.yannickdeturck.akkademo.pizzaorderer.actors.Office

object WorkshopRunner extends App{
  val system = ActorSystem("akka-workshop")
  val office = system.actorOf(Office.props, "office")
  office ! Office.StartWorkshop
  system.awaitTermination()
}

File: intro-to-akka-demo/src/main/java/be/yannickdeturck/akkademo/pizzaorderer/WorkshopRunner.java
package be.yannickdeturck.akkademo.pizzaorderer;

import akka.actor.ActorRef;
import akka.actor.ActorSystem;
import be.yannickdeturck.akkademo.pizzaorderer.actors.Office;

public class WorkshopRunner {
    public static void main(String[] args) {
        ActorSystem system = ActorSystem.create("akka-workshop");
        final ActorRef office = system.actorOf(Office.props(), "office");
        office.tell(new Office.StartWorkshop(), ActorRef.noSender());
        system.awaitTermination();
    }
}
